release-notes/5.0/5.0-known-issues.md

```diff
@@ -16,6 +16,14 @@ This document lists known issues for **.NET 5 Preview 1 and beyond releases** wh
 
 1. The Microsoft.Windows.Compatibility package references dependencies that are not available at the targeted versions. To workaround, individually reference the packages your project needs. It is tracked by [dotnet/runtime/issues #34351](https://github.com/dotnet/runtime/issues/34351)
 
+
+## ASP.NET Core Runtime
+
+### Preview 4
+
+1. The dotnet-certs tool will corrup the dev-certificate if run in any Docker-enabled scenario. If you create / load any docker-enabled ASP.NET Core project the tool will run automatically to check for a valid certificate, won't find it and will try to recreate it. As a result, it'll leave the certificate in a corrupted state.
```

Comment:
```suggestion
1. The dotnet-certs tool will corrupt the dev-certificate if run in any Docker-enabled scenario. If you create / load any docker-enabled ASP.NET Core project the tool will run automatically to check for a valid certificate, won't find it and will try to recreate it. As a result, it'll leave the certificate in a corrupted state.
```

Reply:
Thanks, @martincostello. I'm actually rewriting this section.